readme.md

```diff
@@ -40,7 +40,7 @@ Please read our [guides](https://docs.ignite.com/network/introduction) for using
 
 ## Learn more
 
-Starport Network (SPN) is a free and open source product maintained by [Ignite](https://ignite.com). Here's where you can find us. Stay in touch.
+Ignite Network (SPN) is a free and open source product maintained by [Ignite](https://ignite.com). Here's where you can find us. Stay in touch.
```

Comment:
Do we still want to use SPN?

Reply:
Let's discuss this on the next meeting,because we need to find a new short name for the chain ID and the denom